src/fios_gui.cpp

```diff
@@ -672,6 +793,8 @@ struct SaveLoadWindow : public Window {
 				break;
 
 			case WID_SL_DRIVES_DIRECTORIES_LIST: { // Click the listbox
+				printf("test");
```

Comment:
Things like these can probably be taken out

Reply:
oops